packages/core/src/lib/nav-dropdown/nav-dropdown.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+import { clickOutside } from '$lib/click-outside';
+import { Icon } from '$lib/icon';
+import { createHandleKey } from '$lib/keyboard';
+import { Floating, matchWidth } from '$lib/floating';
+
+interface VersionOption {
+  label: string;
+  detail?: string;
+  description?: string;
+  href: string;
+}
+
+export let options: VersionOption[] = [];
+export let selectedHref: string;
+
+let isOpen = false;
+let activeIndex = -1;
+let buttonElement: HTMLButtonElement;
+
+$: selected =
+  options.find((opt) => opt.href === selectedHref)?.label ?? 'Latest Version';
+
+const toggleDropdown = () => {
+  isOpen = !isOpen;
+};
+
+const closeDropdown = () => {
+  isOpen = false;
+};
+
+const handleKeydown = createHandleKey({
+  Enter: () => {
+    if (!isOpen) {
+      isOpen = true;
+    } else if (activeIndex >= 0) {
+      const selectedOption = options[activeIndex];
+      if (selectedOption) {
+        window.location.href = selectedOption.href;
```

Comment:
I think my confusion around using app/navigation stemmed from trying to navigate to the appropriate href when someone hit the enter key.  Is window.location appropriate here?